src/controllers/propertyInfo/propertyController.js

```diff
@@ -19,10 +19,10 @@ export const createProperty = async (req, res) => {
 };
 
 // eslint-disable-next-line consistent-return
-export const allProperties = async (req, res) => {
+export const getAllProperties = async (req, res) => {
   try {
     const getProperties = await propertyModel.select('*');
-    if (getProperties.rows === 0) {
+    if (getProperties.rows.length === 0) {
       return res.status(409).json({ message: 'no property' });
```

Comment:
I don't think this is the status code to use, 404 is not found also use a more descriptive message like no property found